### ISSUE TITLE
UnityML-agents 하는방법

### ISSUE THREAD

**Issue by kimdn0520** [kimdn0520/UnityML#1]
* anaconda prompt 로 한다 
  안되면 cmd 걍 켜서 해볼것?

* 폴더하나 만들어서
  md python -envs 로 폴더를 만든다

* python -m venv python-envs\sample-env  하면 sample-env 폴더가 생김

* python-envs\sample-env\Scripts\activate // 의존라이브러리 활성화

* pip install --upgrade pip   또는 python -m pip install -upgrade pip

* pip3 install mlagents

* cuda 깔기, cudnn 깔기 (인터넷에 찾아볼것) cudnn 환경변수 설정하는것도 해야함
  cudart64 101.dll 파일 받아서  C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.0\bin
  여기에 넣기! 

* mlagents-learn 으로 다 깔렷는지 확인

* 폴더안에 메모장으로 xxx.yaml  생성
  다른이름으로 저장해서 .yaml형식으로 저장해야댐 ! 
  behaviors:
  RollerBall:
    trainer_type: ppo
    hyperparameters:
      batch_size: 10
      buffer_size: 100
      learning_rate: 3.0e-4
      beta: 5.0e-4
      epsilon: 0.2
      lambd: 0.99
      num_epoch: 3
      learning_rate_schedule: linear
    network_settings:
      normalize: false
      hidden_units: 128
      num_layers: 2
    reward_signals:
      extrinsic:
        gamma: 0.99
        strength: 1.0
    max_steps: 500000
    time_horizon: 64
    summary_freq: 10000

    이런거 해야함!
https://github.com/Unity-Technologies/ml-agents/blob/release_6_docs/docs/Training-Configuration-File.md
여기 참고해서 애들이 뭔지 파악하기 

* mlagents-learn C:\junba\baller.yaml --env=C:\junba\ball --run-id=first
  // yaml이 있는 파일로 위치지정,  ball 은 유니티 실행파일임 위치지정 해준다 
     first는 내가 이름 정해준것 result 폴더안에 first폴더가 생기고 거기에 모델이 저장된다 

* 학습이 실행된다! Ctrl + c 하면 멈춤 !
